### DISCUSSION TITLE
problem with devexpress component and rtlcss !!

### DISCUSSION THREAD

**Discussion by mbtolou** [quasarframework/quasar#11447] [General - Components / Directives / etc]
I created a repository to test the dxdatagrid

There was no way I could match the dxdatagrid and the rtlcss.
Can you help me solve the problem?

https://github.com/mbtolou/devextreme_rtlcss

**Comment by hawkeye64** [comment]
Have you asked on their repo? This doesn't appear to be a quasar issue.

**Reply by mbtolou** [reply]
Not quasar issue.
But how can I ignore rtlcss in div ?

**Reply by hawkeye64** [reply]
It depends if they are using an RTL post-processor, like this one: https://www.npmjs.com/package/postcss-rtl

**Reply by mbtolou** [reply]
Hello
The correct arrangement of the columns is as follows.

![image](https://github.com/quasarframework/quasar/assets/6413705/dba11960-2ccb-48c2-a74e-caa2864da842)

When the library is converted to RTL, the order of the columns gets messed up and several other bugs occur, for example, the scroll bar does not work properly in the right-to-left mode.

Thank you for helping me to solve the problem?

https://github.com/mbtolou/devextreme_rtlcss

![image](https://github.com/quasarframework/quasar/assets/6413705/d8c41928-c2f1-4018-8e09-20712435cc71)

**Comment by pdanpdan** [comment]
~Make a codepen, please~
Sorry, I didn't see it's the repo

**Reply by pdanpdan** [reply]
You have to inspect (css) what the grid does when RTL is enabled

**Reply by mbtolou** [reply]
I checked it. 

dxdatagrid is changed via javascript when page right to left.
probably the only solution is ignore  process the css related to dxdatagrid by rtlcss . 
If this happens the rtl settings of the component itself will work fine And this will solve the problems.

**Reply by pdanpdan** [reply]
Yes, I'm pretty sure the css from grid should not be processed by rtlcss

**Reply by mbtolou** [reply]
Can you solve the problem?
